plugin/adsk/scripts/mayaUsd_createStagesFromExistingLayer.mel

```diff
@@ -36,31 +47,35 @@ global proc stageFromExistingLayer_UISetup(string $parent) {
         -ann "When on, loads all prims marked as payloads. When off, all prims marked as payloads and their children are not loaded." 
         -sbm "Loads prims marked as payloads" 
         loadPayloadsCheckBox;
+
+    return $layout;
```

Comment:
Change the proc to return the layout it's creating so that the option box can attach it correctly.  When this same proc is called from the File Dialog UI it's handled a bit differently and doesn't need it.